src/Phinx/Config/Config.php

```diff
@@ -49,6 +49,11 @@ class Config implements ConfigInterface
     protected $configFilePath;
 
     /**
+     * @var $phinx_version
+     */
+    public static $phinx_version = "0.4.1";
```

Comment:
Nothing can be trusted, this is programming. ;) FWIW, that's just how [Symfony Console Application works](http://api.symfony.com/2.4/Symfony/Component/Console/Application.html#method___construct).